cmd/service_run.go

```diff
@@ -403,7 +403,7 @@ func waitSig(ctx context.Context, cfunc func(), log *zap.Logger, pendingSigReque
 					p.BangMark().DangerText("Error: ").DangerText(sentTx.Error.Error()).NextLine()
 					p.BangMark().DangerText("Details: ").DangerText(strings.Join(sentTx.ErrorDetails, " ,")).NextSection()
 				} else {
-					p.CheckMark().SuccessText("Transaction sent! ").NextLine()
+					p.CheckMark().SuccessText("Transaction sent! ID:").SuccessText(sentTx.TxID).NextLine()
```

Comment:
I don't think we should display it on the CLI. The tx ID is more of an internal property used to consolidate track the transaction internally between the approval and sending steps. It might confuse the user,